### PR TITLE
add clean-text

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
@conda-forge/core someone can help me with this ?

**Review by chrisburr — CHANGES_REQUESTED** [review]
The sdist on PyPI isn't a source distribution as it contains a shared library that was built for Linux `text-clean-0.1.5/text_clean/text_clean.so`. The GitHub repo also doesn't exist so I'm not even sure where it comes from at a glance.

**Comment by FernandezMathieu** [comment]
@chrisburr 
Yes the Pypi librarie page look strange to me too. 
I found the github page here => https://github.com/jfilter/clean-text

**Comment by chrisburr** [comment]
That's a different package, this one is building `text-clean`

**Comment by FernandezMathieu** [comment]
Omg how can i didn't saw that ?

**Comment by chrisburr** [comment]
> Omg how can i didn't saw that. 

I did exactly the same thing when checking where the original error was coming from 😆

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please rerender

File: recipes/clean-text/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - emoji
+    - ftfy >=5.8,<6.0
```
Comment (chrisburr) [inline recipes/clean-text/meta.yaml:24]: This failed to build in master for some reason, can you take a look: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=266046&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d

**Comment by FernandezMathieu** [comment]
@conda-forge-admin restart ci

File: recipes/clean-text/meta.yaml (hunk 1 above)
Comment (FernandezMathieu) [reply recipes/clean-text/meta.yaml:24]: That's weird, pipelines were all goods during PR. It's not supposed to be the same pipelines ?

Comment (chrisburr) [reply recipes/clean-text/meta.yaml:24]: It should be identical, perhaps it's just a flaky test. I'd suggest opening a PR to see if it fails again (ping me on it once you do).

Comment (FernandezMathieu) [reply recipes/clean-text/meta.yaml:24]: Done https://github.com/conda-forge/ftfy-feedstock/pull/16

**Comment by FernandezMathieu** [comment]
@conda-forge-admin restart ci

**Comment by FernandezMathieu** [comment]
Ok i might need advice here @chrisburr

**Comment by chrisburr** [comment]
You just needed to wait longer for the CDN to update and get the new package (it's typically less than an hour after it appears on anaconda.org).

The error is now that `poetry` is missing from `host`:

> ModuleNotFoundError: No module named 'poetry'

**Comment by FernandezMathieu** [comment]
Another one @chrisburr 💯  ! You will merge during the day ?

**Comment by FernandezMathieu** [comment]
@conda-forge/core hello anyone can merge this please ?

**Comment by ocefpaf** [comment]
@chrisburr you requested some changes. Can you take a look and see if everything was addressed as you expected?

**Pull request merged**